egs2/swbd_da/asr1/conf/tuning/train_asr_transformer_postencoder.yaml

```diff
@@ -0,0 +1,34 @@
+# network architecture
+# encoder related
+encoder: transformer
+encoder_conf:
+    output_size: 256    # dimension of attention
+    attention_heads: 4
+    linear_units: 2048  # the number of units of position-wise feed forward
+    num_blocks: 12      # the number of encoder blocks
+    dropout_rate: 0.1
+    positional_dropout_rate: 0.1
+    attention_dropout_rate: 0.0
+    input_layer: conv2d # encoder architecture type
+    normalize_before: true
+
+# postencoder
+postencoder: transformers
+postencoder_conf:
+    model_name_or_path: "bert-base-cased"
+
+# decoder related
+decoder: transformer
+decoder_conf:
+    attention_heads: 4
+    linear_units: 2048
+    num_blocks: 6
+    dropout_rate: 0.1
+    positional_dropout_rate: 0.1
+    self_attention_dropout_rate: 0.0
+    src_attention_dropout_rate: 0.0
+
```

Comment:
I would recommend using Adam optimiser instead of default Adagrad with learning rate of the order of 1e-5. Also warmup scheduler will be very helpful for effective finetuning.

Reply:
Thanks, I'll make another PR with better configurations. This configuration file is just to show how to enable NLU.